paths/uploads/create.yaml

```
@@ -94,6 +94,11 @@ requestBody:
             type: boolean
             default: true
             example:
+          update_translations_on_source_match:
+            description: Pass `true` to update target translations only if the source translations match the uploaded multilingual file
```

I would leave out the "pass `true` wording, as we don't use it anywhre else and for boolean flags this should be clear already.

```suggestion
            description: Update target translations only if the source translations of the uploaded multilingual file match the stored translations
```

Also I think the sentence can include a bit more details for the match logic